spec/turn_spec.rb

```diff
@@ -0,0 +1,226 @@
+                deck2 = Deck.new([card4, card3, card6, card7])
+
+                player1 = Player.new("Megan", deck1)
+                player2 = Player.new("Aurora", deck2)
+
+                turn = Turn.new(player1, player2)
+                turn.pile_cards
+                expect(turn.spoils_of_war).to eq([])
+            end 
+        end
+
+        describe 'award_spoils' do 
+            it ':basic add each of the cards in the spoils_of_war array 
+            to the deck that belongs to the winner of the turn' do 
+                card1 = Card.new(:heart, 'Jack', 11)
+                card2 = Card.new(:heart, '10', 10)
+                card3 = Card.new(:heart, '9', 9)
+                card4 = Card.new(:diamond, 'Jack', 11)
+                card5 = Card.new(:heart, '8', 8)
+                card6 = Card.new(:diamond, '8', 8)
+                card7 = Card.new(:heart, '3', 3)
+                card8 = Card.new(:diamond, '2', 2)
+                cards = [card1, card2, card3, card4, card5, card6, card7, card8]
+
+                deck1 = Deck.new([card1, card2, card5, card8])
+                deck2 = Deck.new([card3, card4, card6, card7])
+        
+                player1 = Player.new("Megan", deck1)
+                player2 = Player.new("Aurora", deck2)
+            
+                turn = Turn.new(player1, player2)
+                turn.pile_cards
+                expect(turn.spoils_of_war).to eq([card1, card3])
+                turn.spoils_of_war
+                turn.award_spoils(player1)
+                
+                expect(player1.deck.cards).to eq([card2, card5, card8, card1, card3])
+            end 
+        
+            it ':war add each of the cards in the spoils_of_war array 
+            to the deck that belongs to the winner of the turn' do
+
+                card1 = Card.new(:heart, 'Jack', 11)
+                card2 = Card.new(:heart, '10', 10)
+                card3 = Card.new(:heart, '9', 9)
+                card4 = Card.new(:diamond, 'Jack', 11)
+                card5 = Card.new(:heart, '8', 8)
+                card6 = Card.new(:diamond, 'Queen', 12)
+                card7 = Card.new(:heart, '3', 3)
+                card8 = Card.new(:diamond, '2', 2)
+                cards = [card1, card2, card3, card4, card5, card6, card7, card8]
+                
+                deck1 = Deck.new([card1, card2, card5, card8])
+                deck2 = Deck.new([card4, card3, card6, card7])
+
+                player1 = Player.new("Megan", deck1)
+                player2 = Player.new("Aurora", deck2)
+
+                turn = Turn.new(player1, player2)
+                turn.pile_cards
+                expect(turn.spoils_of_war).to eq([card1, card4, card2, card3, card5, card6])
+                turn.spoils_of_war
+                turn.award_spoils(player2)
+                
+                expect(player2.deck.cards).to eq([card7, card1, card4, card2, card3, card5, card6])
+            end
+        end
+    end
+
+
+    
+
```

Comment:
I think the indentation in this file is a little "off".

The turn spec was a really thorough one and you hit a lot of the tests with flying colors. There were a few expectations missing from this spec though.

At the start of the spec, we want to make sure to add specs for reading the instance variables of a class. So that would mean:

```ruby
expect(turn.player1).to eq(player1)
expect(turn.player2).to eq(player2)
expect(turn.spoils_of_war).to eq([])
```

After that your tests are great. You have tests for all methods under a `:basic`, `:war`, and `:mutually_assured_destruction` scenario. The only one you seemed to miss was `award_spoils` for a `:mutually_assured_destruction` situation and I believe that by adding this test you would have seen the issue that I walked about before with `"No Winner"`. That test would just make sure that the `@spoils_of_war` instance variable didn't get any cards added to it and that both players had their first 3 cards removed.